spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/SparkSpaceCurveStrategy.java

```diff
@@ -57,8 +57,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class SparkZOrderStrategy extends SparkSortStrategy {
```

Comment:
Maybe we cannot rename the public interfaces. It may break compatibility.
We need to deprecate first and provide an alternate implementation. Later remove the deprecated class in the next version.

Reply:
get it,thanks!